### PR TITLE
Add support for following debug links

### PR DESCRIPTION
**Pull request by d-e-s-o**
Please see individual commits for descriptions.

### CODE REVIEW

File: src/dwarf/resolver.rs [hunk 1]
```diff
@@ -75,15 +81,114 @@ impl From<Option<gimli::DwLang>> for SrcLang {
...
+    return_if_exists!(path);
+
+    let path = Path::new("/usr/lib/debug/").join(file);
+    return_if_exists!(path);
+
+    // Next check others that depend on the absolute `linker` (which may
+    // not be retrievable).
+    // TODO: Different heuristics may be possible here. E.g., honor
+    //       .debug directories and check the current working directory
+    //       (??). Also, users could want to pass in a directory.
```
Comment (anakryiko) [inline src/dwarf/resolver.rs:113]: yep, that seems like another thing that some advanced users would want to "plug into".

**Review by anakryiko — APPROVED** [review]
LGTM, let's add a comment to the parts that try to find "linkee" binary, as it's confusing. But other than that looks good.

File: src/dwarf/resolver.rs (hunk 1 above)
Comment (danielocfb) [reply src/dwarf/resolver.rs:120]: Yeah, will add this as a follow-on.

**Comment by danielocfb** [comment]
> LGTM, let's add a comment to the parts that try to find "linkee" binary, as it's confusing. But other than that looks good.

Added; thanks for the review.

**Pull request merged**